apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/index.vue

```diff
@@ -10,7 +10,7 @@ const cohortOnly = computed(() => {
   const routeSetting = route.query["cohort-only"] as string;
   return routeSetting === "true" || config.public.cohortOnly;
 });
-if (cohortOnly) {
+if (cohortOnly === true) {
   await navigateTo(`/${route.params.schema}/ssr-catalogue/all`);
 }
```

Comment:
should be moved to pre route change hander ( on the server), adding route changes to page loads is messy and results in unneeded redirects

Reply:
https://nuxt.com/docs/guide/directory-structure/middleware